lib/settings_base.py

```diff
@@ -1247,15 +1247,21 @@ def JINJA_CONFIG():
 CSP_REPORT_ONLY = True
 
 CSP_DEFAULT_SRC = ("'self'",)
-CSP_SCRIPT_SRC = (
+CSP_IFRAME_SRC = (
+    "'self'",
+    'https://*.paypal.com',
```

Comment:
Yep - good point. We should try without the wildcard. That would make sense because if we have the wildcard (assuming I'm reading the docs correctly) we don't need the additions to -dev and stage.